tests/test_bai.py

```diff
@@ -68,6 +68,7 @@ def test_author_tokenizer(self):
         token_list = author_tokenize("Ellis Richard")
         self.assertEqual(token_list, ['E Richard',
                                       'Ellis Richard',
+                                      'Richard',
```

Comment:
Can you check if this would make also the case @tsgit pointed out about single word authors? E.g.:
```
873830 ['Ekta']
878180 ['Pushpa']
878351 ['Aprilia']
880819 ['Arianto']
886171 ['Triyanta']
887036 ['Venugopal']
891870 ['Arianto']
892104 ['Abdussattar']
892220 ['Khairurrijal']
893186 ['Nuruzzaman']
893632 ['Abdussattar']
894143 ['Sunil']
899769 ['Sanja']
900162 ['Fayyazuddin']
```
?